tests/behaviour/expectations/behaviour.ts

```diff
@@ -470,6 +470,35 @@ export const expectations = flatten(
               ],
             ),
           ]),
+          File.Simple('static_array_return_index', [
+            new Expect(
+              'testing a static array of ints can be passed into an external function and written to memory and index returned.',
```

Comment:
These descriptive test names are great, but I'm not a fan of the `testA, testB` naming convention, makes it harder to see what's being tested when looking at the contract and makes it easier to accidentally call the wrong function from a test

Reply:
OK sure thing, will change them back.